docs/ruleset_functions/get_number_of_floors.md

```diff
@@ -0,0 +1,54 @@
+# get_number_of_floors
+**Schema Version:** 0.0.23  
+
+**Description:** gets the number of floors in the building - logic is based on looking at the elevation of zones.  Parking Garages are not counted
+
+**Inputs:**
+- **zone_input_list** - the list of the zones for which we want to get the number of floors (changed from RMI to this so that we can get the number of floors for a limited list of zones, for example for the function get_hvac_systems_5_6_serving_multiple_floors_b)
+
+**Returns:**  
+- **result**: an integer indicating the number of floors
+ 
+**Function Call:**
+
+## Logic:
+- make a list of eligible zones in the building: `zone_list = []`
+	- loop through each zone in the input list: `for zone in zone_input_list: `
+		- eligible zones are zones with ceiling height greater than 7.5 (2.286m)': `if zone.ceiling_height > 2.286`
+			- loop throught the spaces in the zone: `for space in zone:`
```

Comment:
is the height check to filter out plenum zones?

Reply:
ASHRAE defines a zone as being at least 7.5' tall, so yes.

Reply:
@supriyagoel, plenums are not supposed to be modeled - see note to the "adjacent_to" data element in the RMD.
@KarenWGard I am not sure why you are using ceilign height criteria. 
There is definition of "story" in 90.1 "story: portion of a building that is between one finished floor level and the next higher finished floor level or the roof, provided, however, that a basement or cellar shall not be considered a story."
I think we should use it as the criteria here. 7.5' height only comes up for the gross floor area definition, so I don't think it applies here. Perhaps we need to add building.number_of_floors